lib/versionmanager.js

```diff
@@ -358,19 +359,17 @@ function queryVersions(packageMap, options = {}) {
 
     // ignore 404 errors from getPackageVersion by having them return null instead of rejecting
     function getPackageVersionProtected(dep) {
-        if (bar) {
-            bar.tick();
-        }
+        bar.tick();
         return getPackageVersion(dep, packageMap[dep], options.pre).catch(err => {
-            if (err && (err.message || err).toString().match(/E404|404 Not Found|doesn't exist/i)) {
+            if (err && (err.message || err).toString().match(/E404|404 Not Found|doesn't exist/i) ||
+                err instanceof packageJson.PackageNotFoundError) {
+                bar.tick();
```

Comment:
You can remove this tick regardless, because after the error is caught it will flow into the next `then` block.